client/flutter/lib/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:intl/intl.dart';
-import './home_page.dart';
+import 'pages/home_page.dart';
```

Comment:
so when its a subdir, you don't use './pages'?  I do not know the convention just asking

Reply:
These were all automatically generated by the IDE, so I am just sticking with its recommendation :) I assume it is following the flutter format convention.